freud/order.pyx

```diff
@@ -1142,10 +1114,9 @@ cdef class RotationalAutocorrelation(Compute):
 
     @Compute._computed_property()
     def ra_array(self):
```

Comment:
Surely there's a better name for this. `order`? `ra`? `rotational_autocorrelation`? `autocorrelation`?

Reply:
Almost certainly, but I figured we'd save renaming for later when we decide how to be more consistent with API across the module.